lambda-function-urls/Makefile

```diff
@@ -0,0 +1,33 @@
+export AWS_ACCESS_KEY_ID ?= test
+export AWS_SECRET_ACCESS_KEY ?= test
+export AWS_DEFAULT_REGION = us-east-1
+
+usage:       ## Show this help
+	@fgrep -h "##" $(MAKEFILE_LIST) | fgrep -v fgrep | sed -e 's/\\$$//' | sed -e 's/##//'
+
+install:     ## Install dependencies
+	@which awslocal || pip install awscli-local
+
+start:
+	localstack start -d
+
+stop:
+	@echo
+	localstack stop
+
+ready:
+	@echo Waiting on the LocalStack container...
+	@localstack wait -t 30 && echo Localstack is ready to use! || (echo Gave up waiting on LocalStack, exiting. && exit 1)
+
+logs:
+	@localstack logs > logs.txt
+
+run:
+	echo "Load the Cloud Pod to inject a Lambda function that scraps GitHub trending page"; \
+	localstack pod load git://localstack/cloud-pods/lambda-function-urls/lambda-url; \
```

Comment:
A small question here: what do you think that's the best way to organize this repo? Initially, my idea was to have all the pods, i.e., the zipped pod contents at the root. Do you have any idea/preference?

Reply:
Having them in the root would make it difficult to link them to their specific documentation, and how they should be used! I prefer to have separate directories for each pod, along with specific documentation or scripts to run them. @whummer Any thoughts?

Reply:
Just a bit of context here. I initially thought to have the pods in the main root to have shorter CLI commands.
In fact, our git shorthand for the CLI is `localstack pod load git://localstack/cloud-pods/<path to the pod>`.

Reply:
Yes, that makes sense. But how would we link them to their specific documentation `README` files, then? From a CLI perspective, it perfectly makes sense. From the sample perspective, its just referencing the path to the pod (hence an extra directory layer) 😄